h/presenters/annotation_json.py

```diff
@@ -64,6 +69,16 @@ def asdict(self):
         return annotation
 
     @property
+    def user_info(self):
```

Comment:
What is the reason for adding this as a property on the `AnnotationResource` rather than adding this attribute via a formatter as we do for eg. moderation info?

Reply:
Pure laziness to be honest. I'll have a look at doing it as a formatter today. Formatters usually have their own cache, but I think it would still be better to reuse the user service cache for this.

At least it will make the presenter nicer.